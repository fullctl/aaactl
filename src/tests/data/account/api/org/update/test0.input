{
  "password": "test",
  "name": "Foo Bar Org",
  "slug": "foobar"
}
