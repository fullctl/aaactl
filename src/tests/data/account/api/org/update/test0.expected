{"data": [{"id": 2, "name": "Foo Bar Org", "is_default": true, "label": "Foo Bar Org", "slug": "foobar", "selected": null, "personal": false, "is_admin": true, "roles": ["Admin", "admin"], "all_roles": ["Admin", "Member", "admin"}], "errors": {}, "profiling": {"queries": 0}}
